lib/fhir_models/bootstrap/generator.rb

```diff
@@ -196,10 +204,11 @@ def generate_class(hierarchy, structure_def, top_level = false)
                 field.binding.delete('extension')
```

Comment:
This isn't part of this PR, but looking into this more I think we'll need a task in the future to look at bindings with extensions (like `Patient.language`)